src/Profile.js

```diff
@@ -0,0 +1,21 @@
+import { Component } from "react";
+
+
+class Profile extends Component () {
```

Comment:
```suggestion
class Profile extends Component {
```
Same problem that I mentioned in the `App` component.